openedx/core/djangoapps/user_authn/views/registration_form.py

```diff
@@ -93,7 +93,7 @@ def contains_url(value):
     """
     Validator method to check whether full name contains url
     """
-    regex = re.findall(r'https|http?://(?:[-\w.]|(?:%[\da-fA-F]{2}))+', value)
+    regex = re.findall(r'https?://(?:[-\w.]|(?:%[\da-fA-F]{2}))*', value)
```

Comment:
Shouldn't we keep `http` too in regex?

Reply:
`https?` will keep both `http` and `https`.